packages/owl-bot/src/copy-code.ts

```diff
@@ -333,45 +355,56 @@ export function copyDirs(
  */
 export async function copyExists(
   octokit: OctokitType,
-  destRepo: GithubRepo,
+  destRepo: AffectedRepo,
   sourceCommitHash: string,
   logger = console
 ): Promise<boolean> {
   // I observed octokit.search.issuesAndPullRequests() not finding recent, open
   // pull requests.  So enumerate them.
-  const owner = destRepo.owner;
-  const repo = destRepo.repo;
+  const owner = destRepo.repo.owner;
+  const repo = destRepo.repo.repo;
   const pulls = await octokit.pulls.list({
     owner,
     repo,
     per_page: 100,
     state: 'all',
   });
-  for (const pull of pulls.data) {
-    const pos: number = pull.body?.indexOf(sourceCommitHash) ?? -1;
-    if (pos >= 0) {
-      logger.info(
-        `Pull request ${pull.number} with ${sourceCommitHash} exists in ${owner}/${repo}.`
-      );
-      return true;
+  const copyTag = copyTagFrom(destRepo.yamlPath, sourceCommitHash);
+
+  // A generic function that finds matches in either pull request or issue
+  // bodies.
+  const findInBodies = (
+    kind: 'Pull request' | 'Issue',
+    response: {data: {number: number; body?: string | null}[]}
+  ): boolean => {
+    for (const issue of response.data) {
+      const copyTagPos = indexOfCopyTagFooter(issue.body ?? '');
```

Comment:
I added this comment at the first mention of copy-tag:

```ts
/**
 * Owl bot must avoid creating duplicate pull requests that copy code from
 * googleapis-gen.  Duplicate pull requests would annoy and confuse library
 * maintainers.
 *
 * To avoid creating duplicate pull requests, owl bot creates a unique id,
 * called a copy tag, for each copy operation.
 *
 * Before multiple .OwlBot.yaml files were permitted in a single repo, the
 * commit hash from googleapis-gen effectively functioned as a unique id
 * for the copy operation.  However, with multiple .OwlBot.yamls in a single
 * library repo, the commit hash from googleapis-gen no longer uniquely
 * identifies the copy operation.  There is potentially one copy operation for
 * each .OwlBot.yaml.  Therefore, Owl bot composes a unique copy tag comprising
 * the commit hash from googleapis-gen and the path to owlBotYaml in the library
 * repo.
 *
 * Before creating a copy-code pull request, Owl Bot first checks if a pull
 * request or issue already exists with the same copy tag.  If one exists, then
 * Owl Bot does not create a second, duplicate pull request.
 */
export function copyTagFrom(owlBotYamlPath: string, sourceCommitHash: string) 
```